symbol_exporter/ast_symbol_extractor.py

```diff
@@ -6,10 +6,19 @@
 
 # Increment when we need the database to be rebuilt (eg adding a new feature)
 NOT_A_DEFAULT_ARG = "~~NOT_A_DEFAULT~~"
+RELATIVE_IMPORT_IDENTIFIER = "~~RELATIVE~~"
 version = "2"  # must be an integer
```

Comment:
```suggestion
version = "3"  # must be an integer
```
We should start over from scratch again!

I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.